File: src/main/java/com/blogspot/nurkiewicz/Foo.java
package com.blogspot.nurkiewicz;

import org.springframework.jdbc.core.JdbcOperations;

import java.util.Date;

/**
 * @author Tomasz Nurkiewicz
 * @since 31.08.11, 21:26
 */
public class Foo {

	private Bar bar;

	private JdbcOperations jdbcOperations;

	public String serverTime() {
		return bar.format(
				(Date)jdbcOperations.queryForObject("SELECT now()", Date.class)
		);
	}

	public void setBar(Bar bar) {
		this.bar = bar;
	}

	public void setJdbcOperations(JdbcOperations jdbcOperations) {
		this.jdbcOperations = jdbcOperations;
	}
}

File: src/main/scala/com/blogspot/nurkiewicz/Foo.scala
package com.blogspot.nurkiewicz

import org.springframework.jdbc.core.JdbcOperations
import org.springframework.stereotype.Service
import java.util.Date
import org.springframework.beans.factory.annotation.Autowired

/**
 * @author Tomasz Nurkiewicz
 * @since 31.08.11, 21:26
 */
@Service
class Foo @Autowired() (bar: Bar, jdbcOperations: JdbcOperations) {

	def serverTime() = bar.format(jdbcOperations.queryForObject("SELECT now()", classOf[Date]))

}